zetaclient/orchestrator/chain_activate.go

```diff
@@ -0,0 +1,248 @@
+package orchestrator
+
+import (
+	"time"
+
+	ethcommon "github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/ethclient"
+
+	btcobserver "github.com/zeta-chain/zetacore/zetaclient/chains/bitcoin/observer"
+	btcrpc "github.com/zeta-chain/zetacore/zetaclient/chains/bitcoin/rpc"
+	btcsigner "github.com/zeta-chain/zetacore/zetaclient/chains/bitcoin/signer"
+	evmobserver "github.com/zeta-chain/zetacore/zetaclient/chains/evm/observer"
+	evmsigner "github.com/zeta-chain/zetacore/zetaclient/chains/evm/signer"
+	"github.com/zeta-chain/zetacore/zetaclient/chains/interfaces"
+	"github.com/zeta-chain/zetacore/zetaclient/common"
+	"github.com/zeta-chain/zetacore/zetaclient/config"
+)
+
+// WatchEnabledChains watches for run-time chain activation and deactivation
+func (oc *Orchestrator) WatchEnabledChains() {
```

Comment:
```suggestion
func (oc *Orchestrator) WatchActivatedChains() {
```
We should use consistent terminology on this

Reply:
https://github.com/zeta-chain/node/pull/2411/commits/e46e0e43cbea110a018ff480fa41639da7956be4